unit_test/test_Tile_kernels.cc

```diff
@@ -1073,7 +1073,7 @@ void test_device_convert_layout(int m, int n)
         if (m == n)
             slate::device::transpose_batch(n, Aarray_dev, lda, batch_count, queue);
```

Comment:
Is there not an `is_conj` flag to transpose_batch?

Reply:
I did the change only on transpose_batch because I am calling it in internal_gecopy.
But, yes for consistency, I will do the change for the other transpose kernels.

Reply:
I pushed new changes with using is_conj for other transpose interfaces.